charts/greenhouse/templates/plugin-thanos.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: greenhouse.sap/v1alpha1
+kind: Plugin
+metadata:
+  name: thanos
+  namespace: {{ .Chart.Name }}
+spec:
+  disabled: false
+  optionValues:
+    - name: testFramework.image.registry
+      value: keppel.global.cloud.sap/ccloud-ghcr-io-mirror
+    - name: thanos.image.repository
+      value: keppel.global.cloud.sap/ccloud-quay-mirror/thanos/thanos
+    - name: thanos.initChownData.image.registry
+      value: keppel.global.cloud.sap/ccloud-dockerhub-mirror
+    - name: thanos.query.autoDownsampling
+      value: true
+    - name: thanos.query.standalone
+      value: false
+    - name: thanos.query.stores
+      value: []
+    - name: thanos.ruler.enabled
+      value: false
+    - name: thanos.store.enabled
+      value: true
+    - name: thanos.compactor.enabled
+      value: true
+  pluginDefinition: thanos
+  releaseNamespace: kube-monitoring
```

Comment:
This should go into the Greenhouse namespace.

Reply:
```suggestion
```